test_framework/scripts/longhorn-setup.sh

```diff
@@ -467,8 +523,8 @@ main(){
     run_longhorn_upgrade_test
     run_longhorn_tests
   else
-    generate_longhorn_yaml_manifest "${TF_VAR_tf_workspace}"
-    install_longhorn_by_manifest "${TF_VAR_tf_workspace}/longhorn.yaml"
+    get_longhorn_chart
+    install_longhorn_by_chart
```

Comment:
Why can't we install Longhorn by manifest?

Reply:
We need to add `enableGoCoverDir`. However, `longhorn.yaml` is also an installation way on official document. Users don't need to have `enableGoCoverDir` in their environment.

Ref: https://github.com/longhorn/longhorn/pull/7367

Reply:
@longhorn/qa We will change the default installation method from `manifest` to `helm`. For your reference.